src/common/example/index.ts

```diff
@@ -42,12 +94,24 @@ export class App extends WidgetBase<WidgetProperties> {
 				options: modules,
 				getOptionValue: (module: any) => module,
 				getOptionLabel: (module: any) => module
-			})
+			}),
+			v('label', [
+				'Use Dojo Theme ',
+				v('input', {
+					styles: {
+						'margin-top': '20px'
+					},
+					type: 'checkbox',
+					onchange: this.themeChange
+				})
+			]),
+			this._module ? w<any>(this._module, { theme: this._theme }) : null
```

Comment:
Nit, but can we do better than the `<any>` cast?

Reply:
Perhaps could be `ThemedMixin(WidgetBase)` but I assume I would have tried that!

Maybe need to actually make a "fake" class for it to be able to be used as a type.

```ts
class WidgetExample extends ThemedMixin(WidgetBase) {}
```